van-gogh-second-page/maria/fonts/sharefonts.net.txt

```diff
@@ -0,0 +1 @@
+Free download fonts at http://sharefonts.net
```

Comment:
I think we don't import these font files anywhere. You could clean it up if you don't need it)